src/embed_tests/TestFinalizer.cs

```diff
@@ -38,6 +38,7 @@ private static bool FullGCCollect()
             catch (NotImplementedException)
             {
                 // Some clr runtime didn't implement GC.WaitForFullGCComplete yet.
+                Thread.Sleep(300);
```

Comment:
I tried to introduce a small pause here to give the GC some time, didn't help, though. Should I remove this change?

Reply:
Yep.

Reply:
Doh, sorry. I'll do that directly on master.

Reply:
No worries. I think it's good you unblocked merging of https://github.com/pythonnet/pythonnet/pull/1287

Reply:
Yeah, that's why I sped this up a bit ;). I've undone the change on master now.